src/vic3_world/states/vic3_state_importer.cpp

```diff
@@ -10,6 +10,12 @@ vic3::StateImporter::StateImporter()
    state_parser_.registerKeyword("country", [this](std::istream& input_stream) {
       owner_number_ = commonItems::getInt(input_stream);
    });
+   state_parser_.registerKeyword("incorporation", [this](std::istream& input_stream) {
+      if (commonItems::getInt(input_stream) == 1)
```

Comment:
0 or 1? Where's progress? So confusing.

Reply:
My test save had no states in process of incorporation. Now fixed.